make.js

```diff
@@ -548,16 +542,24 @@ target.publish = function() {
     run(`nuget3.exe push ${nupkgFile} -Source ${server} -apikey Skyrise`);
 }
 
+
+var agentPluginTasks = ['DownloadPipelineArtifact', 'PublishPipelineArtifact'];
 // used to bump the patch version in task.json files
 target.bump = function() {
     taskList.forEach(function (taskName) {
         var taskJsonPath = path.join(__dirname, 'Tasks', taskName, 'task.json');
         var taskJson = JSON.parse(fs.readFileSync(taskJsonPath));
+
+        // skip agent plugin tasks
+        if(agentPluginTasks.indexOf(taskJson.name) > -1) {
```

Comment:
This is what I mentioned about tasks whose versions need to align with the Agent. We probably want to improve the core issue but for now we need to make sure we don't bump them.